src/components/Utils/order/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as showUpdateOrderError } from './showUpdateOrderError';
+
+export default {};
```

Comment:
for linter

Reply:
maybe just disable a linter in this file by a comment rather than introducing code to avoid warning?